gripmock/go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/tokopedia/gripmock
+
+go 1.15
```

Comment:
this is a bit old - be wary of incompatibilities

Reply:
it is old, this was taken and modified from the original image. Can see if I can bump it and make sure it still works.

Reply:
moved to a later docker image for gripmock that you found :)